bazel/app_chart.bzl

```diff
@@ -134,9 +134,9 @@ def app_chart(
 
     if not values:
         if chart == "cloud":
-            values = "@cloud_robotics//bazel/build_rules/app_chart:values-cloud.yaml"
+            values = "//bazel/build_rules/app_chart:values-cloud.yaml"
         else:
-            values = "@cloud_robotics//bazel/build_rules/app_chart:values-robot.yaml"
+            values = "//bazel/build_rules/app_chart:values-robot.yaml"
```

Comment:
Use `Label()` in `.bzl` files for targets in this repo to resolve correctly when macro is called from a foreign repo.

Reply:
Done.